src/installation/guides/chroot.md

```diff
@@ -0,0 +1,302 @@
+# Installation via chroot (x86 or x86_64)
+
+This guide details the process of manually installing Void via a chroot on an
+x86 or x86_64 PC architecture. It is assumed that you have a familiarity with
+Linux, but not necessarily with installing a Linux system via a chroot. This
+guide can be used to createa a "typical" setup, using a single partition on a
+single SATA/IDE/USB disk. Each step may be modified to create less typical
+setups, such as [full disk encryption](./fde.md).
+
+Void provides two options for bootstrapping the new installation. The **XBPS
```

Comment:
Bootstrapping refers to the process of compiling your whole system (or most of it). In our case, I'd write for " for creating the new installation" or similar.

Reply:
how about "for installing the base distribution"? I've used that wording elsewhere in this guide

Reply:
distribution -> system, perhaps?

Reply:
"Bootstrapping" does not imply compilation to me; it is simply the process of getting something running from nothing. In `xbps-source`, the `bootstrap` command _does_ compile all of the base packages, but there is also `binary-bootstrap` to bootstrap a chroot from prebuilt binary packages.

If there is still resistance to "bootstrapping", I do agree that "for installing the base system" is an appropriate alternative.

Reply:
I don't think "bootstrapping" is incorrect here, but I do agree that it could cause confusion with compiling the whole thing from source. I'm good with "installing the base system"

Reply:
Well, I think I had been in a bootstrapping from zero mood when I made that comment, but after re-reading I feel like bootstrapping fits just fine.